src/gameLogic/generation/ActionGenerator_Server.cpp

```diff
@@ -0,0 +1,34 @@
+//
+// Created by Carolin on 04.05.2020.
+//
+
+#include <datatypes/gameplay/CatAction.hpp>
+#include <util/GameLogicUtils.hpp>
+#include <datatypes/gameplay/JanitorAction.hpp>
+#include "ActionGenerator.hpp"
+
+namespace spy::gameplay {
+    std::shared_ptr<BaseOperation>
+    ActionGenerator::generateExfiltration(const State &/*s*/, const util::UUID &/*damagedCharacter*/) {
+        // TODO implement
+        return std::shared_ptr<BaseOperation>();
```

Comment:
```suggestion
        return nullptr;
```
(solange das noch nicht implementiert ist, dann fällt das schneller auf :D)